rpc/answer.go

```diff
@@ -202,13 +207,20 @@ func (ans *answer) Return(e error) {
 //
 // The caller must be holding onto ans.c.mu and the sender lock.
 // The result's capability table must have been extracted into
-// ans.resultsCapTable before calling sendReturn. Only one of
+// ans.resultCapTable before calling sendReturn. Only one of
 // sendReturn or sendException should be called.
-func (ans *answer) sendReturn(cstates []capnp.ClientState) (releaseList, error) {
+func (ans *answer) sendReturn() (releaseList, error) {
 	ans.pcall = nil
 	ans.flags |= resultsReady
+	// XXX: do we need to check if we've received a finish? if so, the
```

Comment:
The `XXX` suggests that this is a piece of missing implementation.  Is that (still) the case?

Reply:
Good catch, fixed. Also fixed merge conflicts.